### PR TITLE
Add Maniblitz.md

### PR DESCRIPTION
**Pull request by ManiBlitz**
https://rawgit.com/ManiBlitz/ManiBlitz.github.io/git-checkout--b-MDwiki-page/pages/vi/profiles/Maniblitz.md

### CODE REVIEW

**Comment by xyb994** [comment]
@ManiBlitz  Looks good overall :)

 A few small issues:
- there are a few extra lines at the end of the file, keeping thing tidy is aways the goal ;)
- instead of `[img]`, it would be nice to give those images some description if they mean something in the context, otherwise better leave them empty

**Comment by ManiBlitz** [comment]
New rawgit https://rawgit.com/ManiBlitz/ManiBlitz.github.io/git-checkout--b-MDwiki-page/pages/vi/profiles/Maniblitz.md

**Comment by ManiBlitz** [comment]
https://rawgit.com/ManiBlitz/ManiBlitz.github.io/git-checkout--b-MDwiki-page/#!pages/vi/profiles/Maniblitz.md

File: pages/vi/profiles/Maniblitz.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Hey, I am Karl Kevin Tiba Fossoh :us:
-![img](https://www.jwhomes.com/~/media/JW%20Homes/Georgia/Atlanta/Marietta/The%20Quad%20at%20Meeting%20Park/Carousel/Quad_at_Meeting_Park_by_John_Wieland_Marietta_Square.ashx)
-#### :clock5: Marietta, Georgia, U.S.A. | Time Zone: Eastern Daylight Time(EDT)
-###### :computer: Using macOS High Sierra version 10.13.3
-> Yes I am a developer, not an artist
+# Hey, I am Karl Kevin Tiba Fossoh
+![The beautiful city of Marietta, with his timeless park](https://www.jwhomes.com/~/media/JW%20Homes/Georgia/Atlanta/Marietta/The%20Quad%20at%20Meeting%20Park/Carousel/Quad_at_Meeting_Park_by_John_Wieland_Marietta_Square.ashx)
+#### Marietta, Georgia, U.S.A. | Time Zone: Eastern Daylight Time(EDT)
+###### Using macOS High Sierra version 10.13.3
+ > Yes I am a developer, not an artist
```
Comment (sahilph) [inline pages/vi/profiles/Maniblitz.md]: Whitespace is normally referred as a blank line. Hence Add blank Line before and after this so that the quote will render.
Also remove the unnecessary whitespace on line 28.

**Comment by sahilph** [comment]
@ManiBlitz Looking good. Only a few things left to do (as mentioned in the file comments). Also, for a faster response, please post (with link) on gitter.im for any and every changes you make on an issue or a pull request.

**Comment by sahilph** [comment]
This line `Yes I am a developer, not an artist` would have looked a lot better in a block quote 😃 
Anyways i don't mind if you don't 😉 
Rest of the profile looks good 👍
